.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
+
+This directory contains data from the pytest's cache plugin,
+which provides the `--lf` and `--ff` options, as well as the `cache` fixture.
+
+**Do not** commit this to version control.
```

Comment:
I guess you should not commit that to git :) Maybe add the dir to `.gitignore` ?

Reply:
oops! yes, sure

Reply:
done